Documentation/source/objects/FloatingWindow.rst

```diff
@@ -14,102 +16,33 @@ FloatingWindow
     **document** should be a *NSDocument* instance.
 
 
-.. method:: FloatingWindow.getNSWindow()
```

Comment:
deleted parts are inherited from `Window` via `:members:` and `:inherited-members:`.

only `@decorated` methods need to be repeated here, otherwise Sphinx won’t list them.